MigrationV3V4/CSharp/ContosoHelpdeskChatBot-V4NetFramework/ContosoHelpdeskChatBot/Dialogs/RootDialog.cs

```diff
@@ -0,0 +1,130 @@
+﻿namespace ContosoHelpdeskChatBot.Dialogs
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Builder;
+    using Microsoft.Bot.Builder.Dialogs;
+    using Microsoft.Bot.Builder.Dialogs.Choices;
+
+    public class RootDialog : ComponentDialog
+    {
+        private static log4net.ILog logger = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
+        private const string InstallAppOption = "Install Application (install)";
+        private const string ResetPasswordOption = "Reset Password (password)";
+        private const string LocalAdminOption = "Request Local Admin (admin)";
+        private const string GreetMessage = "Welcome to **Contoso Helpdesk Chat Bot**.\n\nI am designed to use with mobile email app, make sure your replies do not contain signatures. \n\nFollowing is what I can help you with, just reply with word in parenthesis:";
```

Comment:
I forgot that this was the stand in for the welcome user message. You can ignore my previous comment about that. :|